libs/community/langchain_community/vectorstores/neo4j_vector.py

```diff
@@ -753,18 +765,14 @@ def create_new_index(self) -> None:
         to create a new vector index in Neo4j.
         """
         index_query = (
-            "CALL db.index.vector.createNodeIndex("
-            "$index_name,"
-            "$node_label,"
-            "$embedding_node_property,"
-            "toInteger($embedding_dimension),"
-            "$similarity_metric )"
+            f"CREATE VECTOR INDEX {self.index_name} "
```

Comment:
we should add `IF NOT EXISTS`

Reply:
I deliberately didnt add it as I want an error here if it already exists as otherwise there could be sth wrong upstream and we wouldnt know

Reply:
added